src/components/Button/Button.js

```diff
@@ -93,14 +95,37 @@ function PrivateButton({
     ''
   )
 
+  const [isDebounced, setIsDebounced] = useState(false)
```

Comment:
Is it possible to get specs (may be easier if it is a hook)?

Reply:
updated specs: https://github.com/getethos/ethos-design-system/pull/381/commits/dea7d6c8344728304db8362c2d0c94f69660b956